autoload/lsp/options.vim

```diff
@@ -72,6 +72,8 @@ export var lspOptions: dict<any> = {
   diagSignInfoTexthl: 'Pmenu',
   diagSignHintText: 'H>',
   diagSignHintTexthl: 'Question',
+  # ShowReferences in a quickfix list instead of a location list`
+  referencesInQf: false
```

Comment:
This option might be named poorly, as it is also used when showing multiple implemention/definitions/etc in a list.

Reply:
What about using the name "useQuickfix"?

Reply:
> What about using the name "useQuickfix"?

We have the option "autoPopulateDiags" which populates the location list, and I want to keep it like that, as then with this new option it's possible to use both.

What about "useQuickfixForLocations"